app/assets/javascripts/views/story_view.js

```diff
@@ -382,6 +383,12 @@ module.exports = FormView.extend({
     storyElement.effect("highlight", {color: 'lightgreen'}, 1500);
   },
 
+  highlightCreatedStories: function () {
+    let storyID = `#story-${ this.model.get('id') }`;
```

Comment:
I think this can be replaced by simply `this.$el.effect("highlight", {color: 'lightgreen'}, 1500);` because it's the id of the view you're inside https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/views/story_view.js#L69